### DISCUSSION TITLE
Are there any programmers here?

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#1215] [General]
How did you all get started in programming? Did you complete any formal college education, or did you take a different path? I'm also curious if there are effective ways to learn programming at home. What resources or strategies would you recommend for someone looking to self-study and build their skills independently?

**Comment by MirkosMangle** [comment]
With the amount of online courses these days, you can definitely learn it at home.
I did three years on college, just cause it was different in my times

**Comment by hernessteve** [comment]
Taking online courses is definitely a good idea, especially if you’re looking to start learning HTML today. The flexibility of online learning allows you to study at your own pace, which is perfect for anyone with a busy schedule. This site covers everything from the basics of HTML to more advanced CSS techniques, making it an excellent starting point for beginners. The clear explanations and practical examples will help you grasp the concepts quickly, and you’ll find yourself building your own projects in no time. Learning HTML is an essential step toward becoming a proficient web developer, and with the right resources, you can set yourself up for success. The skills you acquire will not only enhance your creativity but also improve your career prospects in this increasingly digital world
